src/Octicons.elm

```diff
@@ -41,7 +41,7 @@ module Octicons exposing (..)
 @docs pathIconWithOptions, polygonIconWithOptions, Options, defaultOptions
 
 # Svg Data
-@docs alertPath, arrowDownPolygon, arrowLeftPolygon, arrowRightPolygon, arrowSmallDownPolygon
+@docs alertPath, archiveData arrowDownPolygon, arrowLeftPolygon, arrowRightPolygon, arrowSmallDownPolygon
```

Comment:
should be `archivePath` and add comma.

Test that your changes are valid by building the library and making sure there are no errors.
```bash
elm-make src/Octicons.elm
elm-make --docs foo.json
```